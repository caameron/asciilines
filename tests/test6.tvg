8 8
C 0 0 h 1
A 1 1 v 1
A 2 2 h 1
M 3 3 h 1
E 4 4 v 1
R 5 5 v 1
O 6 6 v 1
N 7 7 h 1
