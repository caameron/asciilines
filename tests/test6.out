C.......
.A......
..A.....
...M....
....E...
.....R..
......O.
.......N
